<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" rowspan="2" colspan="2">
           <widget class="QGraphicsView" name="graphicsView">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QTextBrowser" name="terminal">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QTextBrowser{	
	
	background-color: rgb(0, 0, 0);
	color: rgb(88, 224, 18);
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="command">
          <property name="styleSheet">
           <string notr="true">QLineEdit{			
	background-color: rgb(0, 0, 0);
	color: rgb(88, 224, 18);
}</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="styleSheet">
             <string notr="true">QPushButton{	
		color: rgb(52, 101, 164); 
	border-color: rgb(52, 101, 164);}</string>
            </property>
            <property name="text">
             <string>Terminal View</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="styleSheet">
             <string notr="true">QPushButton{	
		color: rgb(52, 101, 164); 
	border-color: rgb(52, 101, 164);
}</string>
            </property>
            <property name="text">
             <string>Graphic View</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="styleSheet">
                  <string notr="true">QPushButton{	
	color: rgb(52, 101, 164); 
	border-color: rgb(52, 101, 164);
}</string>
                 </property>
                 <property name="text">
                  <string>(RE)START</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pause">
                 <property name="styleSheet">
                  <string notr="true">QPushButton{	
	color: rgb(52, 101, 164); 
	border-color: rgb(52, 101, 164);
}</string>
                 </property>
                 <property name="text">
                  <string>PAUSE</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_2">
                 <property name="styleSheet">
                  <string notr="true">QPushButton{	
	color: rgb(52, 101, 164); 
	border-color: rgb(52, 101, 164);
}</string>
                 </property>
                 <property name="text">
                  <string>AUTO MOVE</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_11">
                 <item>
                  <spacer name="verticalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Preferred</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="styleSheet">
                    <string notr="true">QLabel{	
	color: rgb(52, 101, 164); 
}</string>
                   </property>
                   <property name="text">
                    <string>Energy</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QProgressBar" name="energybar">
                   <property name="styleSheet">
                    <string notr="true">QSpinBox{	
	color:rgb(239, 41, 41);
}</string>
                   </property>
                   <property name="value">
                    <number>100</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_12">
                 <item>
                  <spacer name="verticalSpacer_6">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Preferred</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="styleSheet">
                    <string notr="true">QLabel{	
	color: rgb(52, 101, 164); 
}</string>
                   </property>
                   <property name="text">
                    <string>Health</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QProgressBar" name="healthbar">
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="value">
                    <number>100</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btn_magic">
             <property name="styleSheet">
              <string notr="true">QPushButton{	
	color: rgb(52, 101, 164); 
	border-color: rgb(52, 101, 164);
}</string>
             </property>
             <property name="text">
              <string>MAGIC</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <spacer name="verticalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Preferred</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="styleSheet">
                  <string notr="true">QLabel{	
	color: rgb(52, 101, 164); 
}</string>
                 </property>
                 <property name="text">
                  <string>Enemy Nr.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="EnemyInput">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit
{	
	color: rgb(52, 101, 164); 
}</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <spacer name="verticalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Preferred</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="styleSheet">
                  <string notr="true">QLabel{	
	color: rgb(52, 101, 164); 
}</string>
                 </property>
                 <property name="text">
                  <string>HealthPack Nr.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="HealthPackInput">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit
{	
	color: rgb(52, 101, 164); 
}</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="2" column="0">
                <widget class="QLCDNumber" name="enemyNum">
                 <property name="styleSheet">
                  <string notr="true">QLCDNumber{
	color: rgb(52, 101, 164); 
	border-color: rgb(52, 101, 164);}</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="labelENum">
                 <property name="styleSheet">
                  <string notr="true">QLabel{	
	color: rgb(52, 101, 164); 
}</string>
                 </property>
                 <property name="text">
                  <string>Rest of enemies</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Preferred</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="2" column="0">
                <widget class="QLCDNumber" name="healthPackNum">
                 <property name="styleSheet">
                  <string notr="true">QLCDNumber{
	color: rgb(52, 101, 164); 
	border-color: rgb(52, 101, 164);}</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="labelHNum">
                 <property name="styleSheet">
                  <string notr="true">QLabel{	
	color: rgb(52, 101, 164); 
}</string>
                 </property>
                 <property name="text">
                  <string>Rest of Health Pack</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Preferred</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_9">
               <property name="styleSheet">
                <string notr="true">QLabel{	
	color: rgb(52, 101, 164); 
}</string>
               </property>
               <property name="text">
                <string>Start X</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0" colspan="2">
              <widget class="QPushButton" name="btn_goto">
               <property name="styleSheet">
                <string notr="true">QPushButton{	
	color: rgb(52, 101, 164); 
	border-color: rgb(52, 101, 164);
}</string>
               </property>
               <property name="text">
                <string>GO TO</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="starty">
               <property name="styleSheet">
                <string notr="true">QLineEdit{	
	color: rgb(52, 101, 164); 
}</string>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLineEdit" name="gotox">
               <property name="styleSheet">
                <string notr="true">QLineEdit{	
	color: rgb(52, 101, 164); 
}</string>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLineEdit" name="startx">
               <property name="styleSheet">
                <string notr="true">QLineEdit
{	
	color: rgb(52, 101, 164); 
}</string>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_11">
               <property name="styleSheet">
                <string notr="true">QLabel{	
	color: rgb(52, 101, 164); 
}</string>
               </property>
               <property name="text">
                <string>Goal X</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_10">
               <property name="styleSheet">
                <string notr="true">QLabel{	
	color: rgb(52, 101, 164); 
}</string>
               </property>
               <property name="text">
                <string>Start Y</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="gotoy">
               <property name="styleSheet">
                <string notr="true">QLineEdit{	
	color: rgb(52, 101, 164); 
}</string>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_12">
               <property name="styleSheet">
                <string notr="true">QLabel{	
	color: rgb(52, 101, 164); 
}</string>
               </property>
               <property name="text">
                <string>Goal Y</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="styleSheet">
                  <string notr="true">QLabel{	
	color: rgb(52, 101, 164); 
}</string>
                 </property>
                 <property name="text">
                  <string>A*</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="styleSheet">
                      <string notr="true">QLabel{	
	color: rgb(52, 101, 164); 
}</string>
                     </property>
                     <property name="text">
                      <string>  G factor     </string>
                     </property>
                     <property name="textFormat">
                      <enum>Qt::PlainText</enum>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="g_box">
                     <property name="styleSheet">
                      <string notr="true">QSpinBox{	
	color: rgb(52, 101, 164); 
}</string>
                     </property>
                     <property name="value">
                      <number>1</number>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSlider" name="g_slider">
                     <property name="pageStep">
                      <number>1</number>
                     </property>
                     <property name="value">
                      <number>1</number>
                     </property>
                     <property name="sliderPosition">
                      <number>1</number>
                     </property>
                     <property name="tracking">
                      <bool>false</bool>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="invertedAppearance">
                      <bool>false</bool>
                     </property>
                     <property name="invertedControls">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="styleSheet">
                      <string notr="true">QLabel{	
	color: rgb(52, 101, 164); 
}</string>
                     </property>
                     <property name="text">
                      <string>  H factor     </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="h_box">
                     <property name="styleSheet">
                      <string notr="true">QSpinBox{	
	color: rgb(52, 101, 164); 
}</string>
                     </property>
                     <property name="value">
                      <number>1</number>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSlider" name="h_slider">
                     <property name="pageStep">
                      <number>1</number>
                     </property>
                     <property name="value">
                      <number>1</number>
                     </property>
                     <property name="sliderPosition">
                      <number>1</number>
                     </property>
                     <property name="tracking">
                      <bool>false</bool>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="invertedAppearance">
                      <bool>false</bool>
                     </property>
                     <property name="invertedControls">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="styleSheet">
                  <string notr="true">QLabel{	
	color: rgb(52, 101, 164); 
}</string>
                 </property>
                 <property name="text">
                  <string>  Speed         </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="speed_box">
                 <property name="styleSheet">
                  <string notr="true">QSpinBox{	
	color: rgb(52, 101, 164); 
}</string>
                 </property>
                 <property name="value">
                  <number>1</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="speed_slider_bar">
                 <property name="styleSheet">
                  <string notr="true">QSpinBox{	
	color: rgb(52, 101, 164); 
}</string>
                 </property>
                 <property name="pageStep">
                  <number>1</number>
                 </property>
                 <property name="value">
                  <number>1</number>
                 </property>
                 <property name="sliderPosition">
                  <number>1</number>
                 </property>
                 <property name="tracking">
                  <bool>false</bool>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="invertedAppearance">
                  <bool>false</bool>
                 </property>
                 <property name="invertedControls">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QPushButton" name="save">
               <property name="styleSheet">
                <string notr="true">QPushButton{		
color: rgb(52, 101, 164); 
	border-color: rgb(52, 101, 164);
}</string>
               </property>
               <property name="text">
                <string>SAVE</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="load">
               <property name="styleSheet">
                <string notr="true">QPushButton{	
		color: rgb(52, 101, 164); 
	border-color: rgb(52, 101, 164);
}</string>
               </property>
               <property name="text">
                <string>LOAD</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
